integration-tests/jpa-h2-embedded/src/main/resources/application.properties

```
@@ -1,4 +1,5 @@
 quarkus.datasource.db-kind=h2
-quarkus.datasource.jdbc.url=jdbc:h2:mem:db1;DB_CLOSE_DELAY=-1;LOCK_TIMEOUT=10000
+#Test with H2 writing to actual disk, as it provides a superset coverage over testing in memory:
+quarkus.datasource.jdbc.url=jdbc:h2:/tmp/h2.db
```

Maybe this would be better, in particular on Windows? It seems tests are passing with `/tmp`, but I can't explain why :shrug: 

```suggestion
quarkus.datasource.jdbc.url=jdbc:h2:target/h2.db
```

---

oh Windows.. didn't think about it! thanks :+1: